src/main/java/seedu/address/logic/commands/AddGroupCommand.java

```diff
@@ -25,9 +28,11 @@ public class AddGroupCommand extends Command {
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Adds a new group to the address book. "
             + "Parameters: "
-            + PREFIX_NAME + "NAME\n"
+            + PREFIX_NAME + "NAME "
+            + "[" + PREFIX_TAG + "TAGS]\n"
             + "Example: " + COMMAND_WORD + " "
-            + PREFIX_NAME + "CS2103T T12-1";
+            + PREFIX_NAME + "CS2103T T12-1"
```

Comment:
We don't support dashes in group names

```suggestion
            + PREFIX_NAME + "CS2103T T12"
```